### ISSUE TITLE
iOS: Distinguish certificates and profiles with the same name

### ISSUE THREAD

**Issue by sandrolain** [appcelerator/atom-appcelerator-titanium#91]
I work with many apps, and with many certificate updates and different versions of AdHoc app profiles to which new devices are enabled.

The certificate list shows the different versions of distribution certificates with the same name and can not distinguish between expiration dates. Moreover it happens that by choosing the second certificate it is actually always selected the first one in the list with the same name, and I can not then select the associated distribution profile.

Likewise if I have multiple profiles for the same app it is not possible to understand the version, as only the name is shown and not the UUID code.

**Comment by hansemannn** [comment]
@sandrolain What would you suggest to improve this behavior? Adding the expiration date when selecting certs?

**Comment by sandrolain** [comment]
@hansemannn 
Hello! Yes, the best thing would be to show the expiration date or the UDID together with the name of the certificate and / or distribution profile. If the certificate is identified by name only, I would recommend to distinguish it by the couple expiry date + name.

**Comment by hansemannn** [comment]
See the linked PR :-) Added via #111 ! Will close this ticket once the PR is merged and released.

**Comment by sandrolain** [comment]
Well, because when you manually install an update of a distribution profile the old one is also kept. It would be necessary to use xcode every time to clean and re-download all profiles.

It would be necessary, however, to do something similar but different also for the certificates, as I see in the code that the value of the option is the very name of the certificate.

`<option value={certificate.name}>{certificate.name}</option>`

I happened to work with two different versions of the same certificate with the same identical name, because each one used to have two profiles of two different apps.
If only the name is used to distinguish the certificate I think there is no uniqueness.

**Comment by hansemannn** [comment]
I get that and have been in a similar situation many times. Did you see the linked pull request? It includes an images of how it now looks, including the "(Expires: xx/xx/xx)" flag.

**Comment by sandrolain** [comment]
Hello. Sorry for the delay in the reply. I saw the screenshot, but it shows only the selection of the app provisioning profile and not the developer certificate.
It is possible that two certificates are installed for the same developer but, for example, one expiring and one just created. The certificates in these cases may be necessary both because one enables some provisioning profiles and the other different profiles.
It happened to me with certificates provided by a third-party company for which I developed different app each enabled by a different version of developer certificates with the exact same name.
